src/components/molecules/json-display.tsx

```diff
@@ -2,7 +2,7 @@ import { useColorModeValue } from 'hooks/chakra-hooks'
 
 import { Card, CardBody } from '@/atoms'
 
-export const JSONDisplay = ({ data }: { data: Record<string, unknown> }) => {
+export const JSONDisplay = ({ data }: { data: Record<any, unknown> }) => {
```

Comment:
This gives us a lint warning but seems important that we support any here.

Reply:
@ryanchristo occasionally an `unknown` type will work, however you can just
```tsx
// eslint-disable-next-line @typescript-eslint/no-explicit-any
```
on the line before. Sometimes `any` is needed